packages/acs-ui-common/package.json

```diff
@@ -60,6 +60,12 @@
     "rimraf": "^2.6.2",
     "rollup": "^4.21.2",
     "ts-jest": "^29.1.5",
-    "typescript": "5.4.5"
+    "typescript": "5.4.5",
+    "synckit": "0.8.8"
+  },
+  "overrides": {    
```

Comment:
Could this instead be in the common-versions files? Just that it would allow the change to be kept to one file and, more importantly, allow a comment _why_ it's been added. (otherwise someone will need to git blame and look up this PR to understand why this is present in the package.json)